Website/src/rendering/src/assets/css/layout/buttons.css

```diff
@@ -10,33 +10,32 @@
   box-shadow: 0 0 20px rgba(0, 0, 0, 0.3);
 }
 
-.btn--main {
+.btn-base {
   @apply font-medium text-center inline-block;
 
   padding: 0.5em 1.55em;
   transition: all 0.2s ease-in-out;
 }
 
-.btn--main--round {
+.btn-main {
+  @apply btn-base;
   @apply bg-yellow text-black rounded-md;
   @apply hover:bg-yellow-dark hover:text-white;
   @apply disabled:bg-yellow-dark disabled:cursor-not-allowed;
 }
 
```

Comment:
Oops, I had that fixed in the other refactoring PR (https://github.com/Sitecore/Sitecore.Demo.Edge/pull/397) I will push the fix here as well.

Reply:
No, because we keep both the old and the new classes in the css selector. I've added a comment about this in the other PR. We are also keeping the .btn__area along with the new .btn-area for this same reason.